sbt-app/src/sbt-test/dependency-management/cache-update/build.sbt

```diff
@@ -23,7 +23,7 @@ lazy val root = (project in file("."))
       ),
       version := "0.3.1-SNAPSHOT",
       description := "An HTTP client for Scala with Async Http Client underneath.",
-      licenses := Seq("Apache 2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt")),
+      licenses := Seq(LicenseInfo("Apache 2", uri("http://www.apache.org/licenses/LICENSE-2.0.txt"), None, None)),
```

Comment:
`None, None` here is not nice. Could you add since tag to produce apply with two fields plz?